pytools/HedwigZarrImage.py

```diff
@@ -220,6 +220,9 @@ def neuroglancer_shader_parameters(self, mad_scale=3) -> dict:
             }
 
         if _shader_type == "MultiChannel":
+            if self.ome_info is None:
```

Comment:
Please move this to the begining of the function to "fail early" and avoid the prior compound logic.